src/helpers/consts.js

```diff
@@ -34,6 +34,8 @@ export const CT_TRAINING              = 'TRAINING';
 export const CT_KITEI_MAKOR           = 'KITEI_MAKOR';
 export const CT_PUBLICATION           = 'PUBLICATION';
 export const CT_LELO_MIKUD            = 'LELO_MIKUD';
+export const CT_BLOG_POST             = "BLOG_POST"
```

Comment:
Like in backend PR. There is no content unit of type `blog_post ` or`tweet`. Please choose a less confusing prefix

Reply:
Done